ui/src/pages/project-details/default-filters-form.tsx

```diff
@@ -0,0 +1,216 @@
+import { FormController } from 'components/form/form-controller'
+import {
+  FormActions,
+  FormError,
+  FormRow,
+  FormSection,
+} from 'components/form/layout/layout'
+import { FormConfig } from 'components/form/types'
+import { AddTaxon } from 'components/taxon-search/add-taxon'
+import { TaxonSelect } from 'components/taxon-search/taxon-select'
+import { Project } from 'data-services/models/project'
+import { InputContent } from 'design-system/components/input/input'
+import { CheckIcon, Loader2Icon, XIcon } from 'lucide-react'
+import { Button, Slider } from 'nova-ui-kit'
+import { Fragment } from 'react'
+import { useForm } from 'react-hook-form'
+import { STRING, translate } from 'utils/language'
+import { useFormError } from 'utils/useFormError'
+
+interface DefaultFiltersFormValues {
+  scoreThreshold?: any
+  includeTaxa?: any
+  excludeTaxa?: any
+}
+
+const config: FormConfig = {
+  scoreThreshold: {
+    label: 'Score threshold',
+    description:
+      'Occurrences with a score below this threshold will be exluded by default for the project.',
+  },
+  includeTaxa: {
+    label: 'Include taxa',
+    description: 'This taxa will be included by default for the project.',
+  },
+  excludeTaxa: {
+    label: 'Exclude taxa',
+    description: 'This taxa will be excluded by default for the project.',
+  },
+}
+
+export const DefaultFiltersForm = ({
+  error,
+  isLoading,
+  isSuccess,
+  onSubmit,
+}: {
+  error?: unknown
+  isLoading?: boolean
+  isSuccess?: boolean
+  onSubmit: (data: DefaultFiltersFormValues) => void
+  project: Project
+}) => {
+  const {
+    control,
+    handleSubmit,
+    setError: setFieldError,
+  } = useForm<DefaultFiltersFormValues>({
+    // TODO: Replace default values with stored values for project
```

Comment:
When default settings are returned with the project we should replace the placeholder values. For returned taxa, the type `{ id: number, name: string}[]` is helpful (rather than a flat list of ids).